CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -788,7 +788,8 @@ private void RemoveSourceHook()
 
         private void CreateOffscreenBrowserWhenActualSizeChanged()
         {
-            if (browserCreated || System.ComponentModel.DesignerProperties.GetIsInDesignMode(this))
+            var webBrowserInternal = this as IWebBrowserInternal;
```

Comment:
Rather than cast and check the `HasParent` param every time there is a resize, I think it's better to have a second `if` statement and set `browserCreated = true` if `HasParent = true`. Thoughts?